packages/react-native-gradle-plugin/src/main/kotlin/com/facebook/react/utils/DependencyUtils.kt

```diff
@@ -84,9 +84,18 @@ internal object DependencyUtils {
     }
   }
 
+  // Allows react-native-tvos to get its dependencies correctly with its custom Maven group name
+  fun readGroupString(propertiesFile: File): String {
```

Comment:
Can we merge `readVersionString` and `readGroupString` into a function that returns a `Pair<String, String>` so we'll be reading from file only once?

Reply:
Done